docs/i18n/id/docusaurus-plugin-content-docs/current/README.md

```diff
@@ -13,38 +13,38 @@ slug: /
 
-Finally, the framework is entirely written in TypeScript. The language brings you optional static type-checking along with the latest ECMAScript features. This allows you to detect most silly errors during compilation and improve the quality of your code. It also offers you autocompletion and a well documented API.
+Terakhir, Foal dibuat menggunakan TypeScript. Typescript sangat membantu dalam urusan type-checking dan memberi kita fitur terbaru dari ECMAScript. Kesalahan dalam kode pun dapat terdeteksi lebih dini. Typescript juga memberikan kita fitur auto-komplit dan dokumen API saat bekerja dalam editor.
 
-## Development Policy
+## Kebijakan saat Develop
 
-> *For contributors only.*
+> *Hanya untuk kontributor.*
 
-### Thousands of Tests
+### Ribuan kali Tes
 
-Testing FoalTS is put on a very high priority. Providing a reliable product is really important to us. As of December 2020, the framework is covered by more than 2100 tests.
+Pengetesan Foal senantiasa menjadi prioritas. Sangat penting bagi kami untuk menghadirkan sebuah produk handal yang benar-benar teruji. Per Desember 2020, Foal telah melalui lebih dari 2100 kali tes. 
 
-### Documentation
+### Dokumentasi
 
-New features, no matter what they offer, are useless if they are not well documented. Maintaining complete and quality documentation is key to the framework. If you think something is missing or unclear, feel free to open an issue on Github!
+Fitur baru, apapun itu, kurang bermanfaat bila tidak disertai dengan petunjuk yang baik. Dokumentasi yang lengkap dan berkualitas adalah kunci dari sebuah framework. Sekiranya ada bagian yang terlewat atau kurang jelas tentang Foal, mohon berkenan melaporkannya via Github, dengan senang hati kami terima! 
 
-### Product Stability
+### Stabilitas Produk
 
-Great attention is paid to the stability of the product. You can find out more by consulting our [dependency policy](https://github.com/FoalTS/foal/blob/master/.github/CONTRIBUTING.MD#dependency-policy), [semantic versioning rules](https://github.com/FoalTS/foal/blob/master/.github/CONTRIBUTING.MD#semantic-versioning) and [long-term support policy](https://github.com/FoalTS/foal/blob/master/.github/CONTRIBUTING.MD#long-term-support-policy-and-schedule).
+Kami mencurahkan segenap perhatian atas stabilitas produk. Untuk keterangan lebih lanjut, silakan rujuk [kebijakan independensi](https://github.com/FoalTS/foal/blob/master/.github/CONTRIBUTING.MD#dependency-policy), [aturan versi semantik](https://github.com/FoalTS/foal/blob/master/.github/CONTRIBUTING.MD#semantic-versioning) dan [kebijakan dukungan jangka panjang](https://github.com/FoalTS/foal/blob/master/.github/CONTRIBUTING.MD#long-term-support-policy-and-schedule).
```

Comment:
This PR looks great!

Quick question on the `kebijakan independensi` here. Is this the right translation? By *dependency policy*, I meant to refer to how the framework relies on other packages (its npm dependencies) but not that the framework is independent.

I don't speak Indonesian so I have to apologize up front if this is right translation. Please let me know what you think is best.

Reply:
Ouch.. you're right!
My shameful mistake, it should be `kebijakan dependensi`, not the other way around.

Thanks for spotting it.